### ISSUE TITLE
Builds are stuck and remain in processing state.

### ISSUE THREAD

**Issue by satyasrikantmantha** [Shippable/support#2040]
Hello,

My builds are stuck since morning and were working yesterday. The project id is 55ed2ff81895ca447413527c

Please take a look and respond asap as I am on a critical path here.

Kind regards,
Satya

**Comment by rageshkrishna** [comment]
Hi @satyasrikantmantha. I'm looking in to this. Would it be alright if I try to trigger a few builds on your project to diagnose this?

**Comment by satyasrikantmantha** [comment]
Hello Krishna, 

Yes it is fine by me. 

regards,
Satya

**Comment by manishas** [comment]
@satyasrikantmantha it looks like you're on our old dedicated hosts which we recently replaced with a beta of our new Single Tenant plan.

An email was sent to your email last week informing about this and requesting everyone on dedicated hosts to migrate to the new plan. It will take less than 5 minutes to migrate and the new plan has a lot of bug fixes and improvements we've made to the single tenant dedicated hosts experience. Also, it is free to move to the Single Tenant plan while in Beta.

Do you want me to resend that email to a specific email address? Let me know if you have any questions.

**Comment by manishas** [comment]
@satyasrikantmantha i have sent the email to your live.in email address in your github profile as well. Please let me know if you have questions... happy to jump on a call and clarify any questions as well.

**Comment by kepkin** [comment]
I have absolutely the same issue. Could you please check my account too?
I use bitbucket with login kepkin and team itkaboom

**Comment by satyasrikantmantha** [comment]
We followed instructions from support above and followed steps to migrate the Dedicated Hosts to Single Tenant CI as mentioned below in circulars sent earlier by shippable. Hope this helps.

"As you know, we introduced the Dedicated Hosts BETA last year to let customers like yourself run builds on their own dedicated hardware over which they have complete control. 
We have received excellent feedback over the last year and have been hard at work hardening the platform and solving any issues that arose. As a result, we are now very close to launching a production ready version of Dedicated Hosts - now rebranded as Single Tenant CI. Single Tenant CI is currently in beta and will be launched broadly in January 2016. Our existing dedicated hosts will be deprecated on 11/14/2015.
To continue uninterrupted service, you will need to move your Dedicated Hosts over to the new Single Tenant Beta subscription before Saturday 11/14/2015. This should not take more than 10 mins.  
The following actions will move your subscription to Single Tenant CI subscription:
Delete your existing dedicated nodes from your existing subscription
Change your Shippable CI subscription to our new single-tenant CI subscription (st-ci-beta)
Re-add your dedicated nodes to the new subscription
You can find step-by-step instructions with screenshots below for performing these actions. These steps should take no more than 5-15 minutes.
Additional information worth knowing:
Changing to the new beta plan will continue to not incur any charges while this remains in beta.
When setting up your new plan, you'll be asked to specify the number of build minions you'd like available to run in parallel within your subscription. In the current beta, this wasn't working properly so the number of builds running concurrently was unconstrained, resulting in build minions receiving insufficient build resources if a large number of builds were running. This has been fixed, so you will now see builds queue properly after the minion count is reached. If you need assistance calculating the best number of minions for your org, please contact me to discuss.
If you cannot perform the actions above by Saturday, November 14th, or if you have any questions any questions for me, please do not hesitate to contact me directly."

**Comment by kepkin** [comment]
I have moved to Single Tenant plan. But it requires me to add *node* before I can run builds. But I don't have node for this, I want to keep running my builds on shippable infrastructure. But now I can't choose shippable node:
![Scrrenshot](http://i.imgur.com/a19N0aul.jpg)

**Comment by rageshkrishna** [comment]
@kepkin Sorry about the confusion on this. If you weren't using dedicated hosts before, you do not need to move to the single tenant plan. To continue running builds on our multi-tenant infrastructure, please move back to the mt-ci plan.

We had an issue from today's service upgrade that caused some subscriptions to run in to this issue of builds staying queued. The issue is being fixed now and your subscription should not run in to this anymore. Please switch back to the mt-ci plan and let me know if builds still fail to trigger for you.

**Comment by kepkin** [comment]
@rageshkrishna 
It works now. Thanks!
